tests/bazeltest.py

```diff
@@ -20,18 +20,12 @@ def test_parse_maven_install(self):
             f.write(MVN_INSTALL_JSON_CONTENT)
 
         result = bazel.parse_maven_install("maven", path)
-
-        self.assertEqual(1, len(result))
+        self.assertEqual(3, len(result))
         dep, transitives, exclusions = result[0]
         self.assertEqual(dependency.new_dep_from_maven_art_str("ch.qos.logback:logback-classic:1.2.3", "maven"), dep)
         self.assertEqual(2, len(transitives))
         self.assertEqual(dependency.new_dep_from_maven_art_str("ch.qos.logback:logback-core:1.2.3", "maven"), transitives[0])
         self.assertEqual(dependency.new_dep_from_maven_art_str("org.slf4j:slf4j-api:jar:1.7.30", "maven"), transitives[1])
-        self.assertEqual(2, len(exclusions))
-        self.assertEqual("jakarta.el", exclusions[0].group_id)
-        self.assertEqual("jakarta.el-api", exclusions[0].artifact_id)
-        self.assertEqual("org.glassfish", exclusions[1].group_id)
-        self.assertEqual("jakarta.el", exclusions[1].artifact_id)
```

Comment:
I need to double check the changes here to ensure it doesn't actually impact the behaviour of pomgen.